.github/workflows/platform_tests.yml

```
@@ -0,0 +1,47 @@
+name: Platform Tests
+on: [push, pull_request]
+
+jobs:
+  platform_tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        go-version: [1.12.x, 1.14.x, 1.15.x]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+
```

To get just 1.12 legacy test run once the following might work:
```suggestion
    exclude:
      - go-version: 1.12.x
        os: [windows-latest, macos-latest]
```

Note: not tested!

---

Nice idea. Will give it a tests :)

---

As discussed earlier, we now run only on latest and newsest.